### DISCUSSION TITLE
Is CircleCi not taking into account the retries set in cypress.json?

### DISCUSSION THREAD

**Discussion by melibe23** [cypress-io/cypress#16301] [CI setup] [answered]
I have a constant fail in circle ci because the first attempt of the test always fail (I intercept a call and happens like this, sorry), but when I run it with run or open, since I have a retry of 2 in cypress.json, it passes. Circle CI is not doing this retry, how can I fix this?

Thanks in advance!

**Comment by bahmutov** [comment, marked as answer]
Can you please provide more details: cypress.json, circleci file, output from the circle run?

**Reply by melibe23** [reply]
Hello, thank you for answering, this is now fixed on CircleCi I can't be sure what happened. Really sorry.

**Reply by bahmutov** [reply]
No worries, glad it is working now
